docs/metricsoverview.rst

```diff
@@ -101,3 +101,8 @@ Generally, if you follow these rules, your tags should work across all backends:
    tag names.
 2. Tag names should begin with a letter.
 3. Tag names should be short.
+4. Tag value domains should be finite and small-ish like under 1,000. Be wary of
```

Comment:
"Tag value domains" is making this kinda hard for me to read. Do you mean that you should only have like under 1000 different tag values?

Maybe "Tag values should be limited to a small (under 1000) set of possible values."?

Reply:
I like that wording a lot better--thank you!